cosmwasm/utils.js

```diff
@@ -675,6 +686,20 @@ const getParameterChangeParams = ({ title, description, changes }) => ({
     })),
 });
 
+const getMigrateContractParams = (config, options, chainName) => {
+    const { contractName, msg } = options;
+
+    const contractConfig = getAmplifierContractConfig(config, contractName);
+    const chainConfig = getChainConfig(config, chainName);
+
+    return {
+        ...getSubmitProposalParams(options),
+        contract: contractConfig[chainConfig?.axelarId]?.address || contractConfig.address,
```

Comment:
We should clean this up soon, having to constantly handle chain specific vs generic config is pretty confusing